packaging/osx/sharedhost/scripts/postinstall

```diff
@@ -11,6 +11,8 @@ INSTALL_DESTINATION=$2
 chmod 755 $INSTALL_DESTINATION/dotnet
 
 # Add the installation directory to the system-wide paths
+# But first create the directory if it doesn't exist
+mkdir -p /etc/paths.d
```

Comment:
Does the installer get looked up in the path with this fix? I ask to understand why Seirra does not have this folder by default while previous OSX did?